src/Sulu/Bundle/WebsiteBundle/Resolver/ParameterResolver.php

```diff
@@ -101,12 +101,14 @@ public function resolve(
         }
         $localizations = [];
 
+        $scheme = $this->requestStack->getCurrentRequest()->getScheme();
```

Comment:
Why aren't we doing that in the `WebspaceManager`? Otherwise the same error might occur in other places as well. And there is a test missing :wink:

Reply:
then the "WebspaceManager" depends on the request - do we want that?

Reply:
The webspace already has the `currentRequest` in sulu 2.0 we already us it for the port see https://github.com/sulu/sulu/pull/5374. So I would fix this in the WebspaceManager and so target same as we targeted for the port fix the `release/2.0` branch.

Reply:
Think the `defaultScheme` should first look at the currentRequest: https://github.com/sulu/sulu/blob/c17ca6b5e366f2ca6e541240a9eecb62590461e2/src/Sulu/Component/Webspace/Manager/WebspaceManager.php#L471 before falling back to the default scheme.